### DISCUSSION TITLE
Linux AppService without Azure DNS

### DISCUSSION THREAD

**Discussion by pettys** [shibayan/appservice-acmebot#259] [Q&A] [answered]
Not sure if this is the right place for a question, but thought I'd throw it out there:

I'd like to add SSL to a linux appservice accessed with a subdomain (eg, myapp.contoso.com) but the domain's DNS is (unfortunately) controlled by GoDaddy and I'm not in a position to get that switched to Azure DNS presently.

If I'm reading this bullet point under "Feature Support correctly, acmebot will _not_ work for me:

* Azure Web Apps (Linux) / Web App for Containers (Windows and Linux, requires Azure DNS)

Is that correct, or am I missing something? It's a bummer to not use this awesome tool because of some of the particulars of this setup. Is there another way to use LetsEncrypt on a setup like this?

**Comment by shibayan** [comment, marked as answer]
Unfortunately, since the Linux App Service needs to be verified using DNS, Azure DNS is required.

A similar solution is available in the Key Vault version of Acmebot, which supports non-Azure DNS providers. It supports non-Azure DNS providers as well.
https://github.com/shibayan/keyvault-acmebot

GoDaddy is not supported, but DNS providers are built to be extensible, so you may be able to support it.

**Reply by pettys** [reply]
Thanks for the response, I really appreciate your time.
